plugins/ledger/ledger.py

```diff
@@ -96,16 +96,13 @@ def is_pin_validated(self, client):
                 return True
             raise e
 
-    def supports_multi_output(self):
-        return self.multiOutputSupported
-
     def perform_hw1_preflight(self):
         try:
             firmwareInfo = self.dongleObject.getFirmwareVersion()
             firmware = firmwareInfo['version'].split(".")
-            self.multiOutputSupported = int(firmware[0]) >= 1 and int(firmware[1]) >= 1 and int(firmware[2]) >= 4
+            bitcoinCashSupport = int(firmware[0]) >= 1 and int(firmware[1]) >= 1 and int(firmware[2]) >= 8
```

Comment:
what does the change from `4` to `8` signify? might be worth pulling this number out into a named variable for readability 👍

Reply:
AFAIK, that's the version for the Bitcoin Cash app in the Nano S. The current version is 1.1.8 @bderose

Reply:
Correct, however the current version checking is a bit odd (ie. 1.1.9 would work but not 1.2.0), so a few things could be tweaked to provide backwards compatibility. Thoughts?

Reply:
One solution would be to use something like the [`semver`](https://pypi.python.org/pypi/semver) package to do this comparison. I realize we may not want to introduce new dependencies, though. In that case, this [SO question](https://stackoverflow.com/questions/11887762/compare-version-strings-in-python) has some good suggestions.

Reply:
Ok, I've added the version comparison from `distutils.version` so no new dependencies are required, and it seems to work well.